README.rst

```diff
@@ -815,6 +832,23 @@ For more information about optional parameters check the
 It can be used to change the thresholds used by `TimeLoggingTestRunner <#openwisp_utilsteststimeloggingtestrunner>`_
 to detect slow tests (0.3s by default) and highlight the slowest ones (1s by default) amongst them.
 
+``OPENWISP_STATICFILES_VERSIONED_EXCLUDE``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**default**: ``[]``
```

Comment:
@pandafy weren't we excluding the leafelet pngs by default?

Reply:
Yes we are in ansible-openwisp2. But should we exclude it here, or exclude it in openwisp-controller?

Reply:
I guess we can exclude it here to keep things simple (with an explanation on why this is being done), can you do that @Arsh0023 please?

Reply:
yep sure @nemesisdesign @pandafy but can you please help me with why we are excluding leaflet pngs

Shall I go with something like - 
Leaflet pngs are excluded by default because of a bug that made it incapable to process versioned static files
created by `ManifestFilesMixin`

I got this from [#232](https://github.com/openwisp/ansible-openwisp2/commit/d4e05cf502db05c0f48370c37db43585ce39f270)

Reply:
oh right so we're excluding all the PNGs, that's fine. The explanation can be that we do it to avoid bugs like https://github.com/openwisp/ansible-openwisp2/issues/232. THank you @Arsh0023!

Reply:
thank you @nemesisdesign but should i exclude all the pngs or only the leaflet ones ?

Reply:
We need to exclude only leaflet PNGs. 

@nemesisdesign did you get confused by the example?

Reply:
okay @pandafy on it 👍